fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/AbstractMaterializedViewRule.java

```diff
@@ -80,40 +100,51 @@ protected List<Plan> rewrite(Plan queryPlan, CascadesContext cascadesContext) {
             if (queryPlan.getGroupExpression().isPresent()
                     && materializationContext.alreadyRewrite(
                     queryPlan.getGroupExpression().get().getOwnerGroup().getGroupId())) {
+                logger.info(currentClassName + " this group is already rewritten so skip");
                 continue;
             }
-            Plan mvPlan = materializationContext.getMtmv().getCache().getLogicalPlan();
-            List<StructInfo> viewStructInfos = extractStructInfo(mvPlan, cascadesContext);
+            MTMV mtmv = materializationContext.getMtmv();
```

Comment:
some function use `Mtmv` and others use `MTMV`. we need to unify them

Reply:
have fixed it